src/components/Differential/ScatterPlotDiv.jsx

```diff
@@ -86,18 +87,33 @@ class ScatterPlotDiv extends Component {
       // user closes volcano plot, close options
       this.setState({ optionsOpen: false, usageOpen: false });
     }
+    if (
+      differentialResultsTableStreaming !==
+        prevProps.differentialResultsTableStreaming ||
+      // need this check for cached data from another test
```

Comment:
the prop for streaming will always change, except for cached data (returning to a test you previously loaded), when we need to check for this test change